### PR TITLE
Scholarships announcement

### PR DESCRIPTION
**Pull request by flaki**

### CODE REVIEW

File: _posts/2019-08-19-this-week-in-rustfest-barcelona-scholarships.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+---
+layout: post
+title: "This Week in RustFest - Scholarships, Community Tickets and CFPs"
+description: "Announcing our Scholarship program, a CFP-extension & a one last round of Community Tickets"
+socialImageSrc: "/assets/posts/barcelona/barcelona-announcement.png"
```
Comment (badboy) [inline _posts/2019-08-19-this-week-in-rustfest-barcelona-scholarships.md:5]: that image 💖

File: _posts/2019-08-19-this-week-in-rustfest-barcelona-scholarships.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  <a href="https://barcelona.rustfest.eu/scholarships/" style="opacity: 1">
+    Scholarship Applications for Barcelona now open!
+  </a>
+</h3>
+
+In our Scholarship program we are offering free tickets and travel support to members of underrepresented groups and people who would otherwise not be able to attend the conference. Based on the experience gathered during previous years' scholarship programs, we had many ideas for improvements of the process, which sadly delayed opening the application form a couple weeks, but there is still plenty of time to apply and we will start awarding the scholarships on a rolling basis in early September.
+
+Many of this year's improvements are centered around making the application process itself more inclusive, as part of this effort we are proud to announce our Scholarships are available in 3 languages: english, [spanish](https://barcelona.rustfest.eu/scholarships/es/) and [catalan](https://barcelona.rustfest.eu/scholarships/ca/)! Read all the other details on the [Scholarships page](https://barcelona.rustfest.eu/scholarships/) and send us feedback, we'd love to hear your feedback and ideas how to further improve our process!
+
```
Comment (badboy) [inline _posts/2019-08-19-this-week-in-rustfest-barcelona-scholarships.md:23]: Add one more sentence to also apply here?

File: _posts/2019-08-19-this-week-in-rustfest-barcelona-scholarships.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+Please note, this is the final round of Community Tickets, there won't be any more rounds and no waitlist will be made available after this round is sold out. Our full price tickets will remain on sale until we reach venue capacity. Just a small reminder, if you can afford them, please opt for the Full Price tickets, supporting us, other ticket holders and our above-mentioned Scholarship (as 10% of all Full Price tickets goes straight into our Scholarship fund).
+
+
+## Extending the CFP deadline
+
+We have alredy received a ton of interesting submissions to our Call for Proposals (thank you!), but due to the raging summer and vacation season we have decided to give you all a bit more time to cook up a proposal: we are extending the CFP with an extra week!
```
Comment (badboy) [inline _posts/2019-08-19-this-week-in-rustfest-barcelona-scholarships.md]: Add the exact date here when it closes.

File: _posts/2019-08-19-this-week-in-rustfest-barcelona-scholarships.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+[Head on to our CFP page to submit a proposal](http://cfp.rustfest.eu/events/rustfest-barcelona-2019), and please [help us spread the word](https://twitter.com/RustFest/status/1153566837386481664), to ensure our [CFP commitee](https://barcelona.rustfest.eu/cfp-committee/) has a real tough time deciding which excellent talk to showcase on the RustFest Barcelona stage this November!
```
Comment (badboy) [inline _posts/2019-08-19-this-week-in-rustfest-barcelona-scholarships.md]: the cfp committee site is currently empty, not sure if we should link to it.

**Review by badboy — CHANGES_REQUESTED** [review]

File: _posts/2019-08-19-this-week-in-rustfest-barcelona-scholarships.md (hunk 4 above)
Comment (apiraino) [inline _posts/2019-08-19-this-week-in-rustfest-barcelona-scholarships.md]:
```suggestion
[Head on to our CFP page to submit a proposal](https://cfp.rustfest.eu/events/rustfest-barcelona-2019), and please [help us spread the word](https://twitter.com/RustFest/status/1153566837386481664), to ensure our [CFP commitee](https://barcelona.rustfest.eu/cfp-committee/) has a real tough time deciding which excellent talk to showcase on the RustFest Barcelona stage this November!
```

Comment (flaki) [reply _posts/2019-08-19-this-week-in-rustfest-barcelona-scholarships.md]: SO YOU SHOULD FIX IT

(yeah I realized that, we should get that populated, but no harm in linking to it IMHO)

File: _posts/2019-08-19-this-week-in-rustfest-barcelona-scholarships.md (hunk 2 above)
Comment (flaki) [reply _posts/2019-08-19-this-week-in-rustfest-barcelona-scholarships.md:23]: Nah, there are plenty of links/CTAs in there already.

**Pull request merged**